config/v0.3.0/config_yaml.go

```diff
@@ -30,6 +31,27 @@ func (j *AttributeNameValueType) UnmarshalYAML(unmarshal func(interface{}) error
 	return nil
 }
 
+// UnmarshalYAML implements json.Unmarshaler.
+func (j *NameStringValuePair) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	var raw map[string]interface{}
+	if err := unmarshal(&raw); err != nil {
+		return err
+	}
+	if _, ok := raw["name"]; raw != nil && !ok {
+		return errors.New("yaml: cannot unmarshal field name in NameStringValuePair required")
+	}
+	if _, ok := raw["value"]; raw != nil && !ok {
```

Comment:
Why check `raw != nil`? If `raw == nil` then `ok` will always be `false` and this will evaluate to `true` with only `!ok`.

Reply:
Fixed